cirq-google/cirq_google/experimental/ops/coupler_pulse.py

```diff
@@ -55,6 +57,9 @@ def __init__(
         coupling_mhz: cirq.TParamVal,
         rise_time: Optional[cirq.Duration] = cirq.Duration(nanos=8),
         padding_time: Optional[cirq.Duration] = cirq.Duration(nanos=2.5),
+        q0_detune_MHz: Optional[cirq.TParamVal]=0.0,
+        q1_detune_MHz: Optional[cirq.TParamVal]=0.0,
```

Comment:
These probably don't need to be `Optional`, can just leave 0.0 as the default value:
```suggestion
        q0_detune_MHz: cirq.TParamVal = 0.0,
        q1_detune_MHz: cirq.TParamVal = 0.0,
```
I'm not sure why we allow `None` for `rise_time` and `padding_time` since both have default values and `None` doesn't have a special meaning.